### DISCUSSION TITLE
Git LFS Project Disk Usage Monitoring

### DISCUSSION THREAD

**Discussion by BenAigan** [git-lfs/git-lfs#4550] [General] [answered]
Is there any way to set quotas on usage per project? 

Also, Is there a way to identify the back end objects from the project without actually downloading the project?

**Comment by bk2204** [comment, marked as answer]
Hey,

The Git LFS client doesn't have this functionality because the protocol doesn't provide it.  Some server implementations may support it, though.  Also, if you know the objects, then you can query them via the protocol, but to enumerate them, you require the repository to walk the pointer files, since the protocol doesn't provide that information, either.

**Reply by BenAigan** [reply]
Was able to determine the Git LFS usage of repositories in Bitbucket by getting the hierarchy value from the repository-config in the repository directory which is the directory name where the LFS objects are uploaded to.
